### PR TITLE
Update the runtime image with the commit: 8bda2fa on main

### PR DESCRIPTION
**Pull request by atheo89**
Update the runtime image with the commit: 8bda2fa on main

### CODE REVIEW

**Review by jstourac — COMMENTED** [review]
`modh` is probably another new organisation in quay.io for me. Do we have it documented somewhere which quay orgs serves to what? :upside_down_face: 

Also, links to `opendatahub/workbench-images:runtime*` in these Dockerfiles are okay? Namely the values of `io.openshift.build.image`, `io.openshift.build.source-location` and `authoritative-source-url`

* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/datascience/ubi8-python-3.8/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/datascience/ubi9-python-3.9/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/minimal/ubi8-python-3.8/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/minimal/ubi9-python-3.9/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/pytorch/ubi8-python-3.8/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/pytorch/ubi9-python-3.9/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/tensorflow/ubi8-python-3.8/Dockerfile
* https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/tensorflow/ubi9-python-3.9/Dockerfile

Regarding the Tensorflow - I don't like that the naming here doesn't match convention with the others - the `runtime` keyword isn't first. Is it possible to change this paradigm?

Doesn't need to be addressed in this PR, we can create an issue for that to track it if we agree.

File: jupyter/datascience/anaconda-python-3.8/runtime-images/ubi8-py38.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "metadata": {
         "tags": [],
         "display_name": "Python 3.8 (UBI8)",
-        "image_name": "quay.io/opendatahub/workbench-images:runtime-minimal-ubi8-python-3.8-6a6098d"
+        "image_name": "quay.io/modh/runtime-images@sha256:a3ee8b8eff99e9699fba1c1a51a9eedc4499caceeb4106e708da048ea0c30ef3"
```
[the remark this answers is not in the document]
Comment (harshad16) [reply jupyter/datascience/anaconda-python-3.8/runtime-images/ubi8-py38.json:6]: yes this is intentional 
these are runtime image, directly doesn't have conda in them
they would be used by elyra pipeline, if someone executes them.

**Comment by harshad16** [comment]
> `modh` is probably another new organisation in quay.io for me. Do we have it documented somewhere which quay orgs serves to what? 🙃
> 
> Also, links to `opendatahub/workbench-images:runtime*` in these Dockerfiles are okay? Namely the values of `io.openshift.build.image`, `io.openshift.build.source-location` and `authoritative-source-url`
> 
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/datascience/ubi8-python-3.8/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/datascience/ubi9-python-3.9/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/minimal/ubi8-python-3.8/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/minimal/ubi9-python-3.9/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/pytorch/ubi8-python-3.8/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/pytorch/ubi9-python-3.9/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/tensorflow/ubi8-python-3.8/Dockerfile
> * https://github.com/red-hat-data-services/notebooks/blob/main/runtimes/tensorflow/ubi9-python-3.9/Dockerfile
> 
> Regarding the Tensorflow - I don't like that the naming here doesn't match convention with the others - the `runtime` keyword isn't first. Is it possible to change this paradigm?
> 
> Doesn't need to be addressed in this PR, we can create an issue for that to track it if we agree.


Documenting about the images, yes we need to do it 
lets improve on that.

about the Dockerfiles labels. we should improve those as well.
will include it as a task.

Naming convention, i wouldn't change it anymore, as it is only on dev side, and users would see display name.

**Comment by harshad16** [comment]
/lgtm
/approve
thanks :+1:
